src/journal/journal-file.c

```diff
@@ -3906,6 +3908,7 @@ void journal_default_metrics(JournalMetrics *m, int fd) {
         char a[FORMAT_BYTES_MAX], b[FORMAT_BYTES_MAX], c[FORMAT_BYTES_MAX], d[FORMAT_BYTES_MAX], e[FORMAT_BYTES_MAX];
         struct statvfs ss;
         uint64_t fs_size = 0;
+        int v;
```

Comment:
style nit: variables should be declared in the inner-most scope - ie, move this inside the "if" block where it's used

Reply:
OK, thanks.